monai/networks/nets/ahnet.py

```diff
@@ -25,29 +25,37 @@ class Bottleneck3x3x1(nn.Module):
 
     def __init__(
         self,
+        spatial_dims: int,
         inplanes: int,
         planes: int,
-        stride: Union[int, Tuple[int, int, int]] = 1,
+        stride: Union[Sequence[int], int] = 1,
         downsample: Optional[nn.Sequential] = None,
     ) -> None:
 
         super(Bottleneck3x3x1, self).__init__()
 
-        conv3d_type: Type[nn.Conv3d] = Conv[Conv.CONV, 3]
-        norm3d_type: Type[nn.BatchNorm3d] = Norm[Norm.BATCH, 3]
+        conv_type = Conv[Conv.CONV, spatial_dims]
+        norm_type: Type[Union[nn.BatchNorm2d, nn.BatchNorm3d]] = Norm[Norm.BATCH, spatial_dims]
+        pool_type: Type[Union[nn.MaxPool2d, nn.MaxPool3d]] = Pool[Pool.MAX, spatial_dims]
         relu_type: Type[nn.ReLU] = Act[Act.RELU]
-        pool3d_type: Type[nn.MaxPool3d] = Pool[Pool.MAX, 3]
-
-        self.conv1 = conv3d_type(inplanes, planes, kernel_size=1, bias=False)
-        self.bn1 = norm3d_type(planes)
-        self.conv2 = conv3d_type(planes, planes, kernel_size=(3, 3, 1), stride=stride, padding=(1, 1, 0), bias=False,)
-        self.bn2 = norm3d_type(planes)
-        self.conv3 = conv3d_type(planes, planes * 4, kernel_size=1, bias=False)
-        self.bn3 = norm3d_type(planes * 4)
+
+        self.conv1 = conv_type(inplanes, planes, kernel_size=1, bias=False)
+        self.bn1 = norm_type(planes)
+        self.conv2 = conv_type(
+            planes,
+            planes,
+            kernel_size=(3, 3, 1)[3 - spatial_dims :],
```

Comment:
this could be `(3, 3, 1)[-spatial_dims:]`

Reply:
thanks, I change them.